tool/speed.cc

```diff
@@ -397,7 +397,7 @@ static bool SpeedRSAKeyGen(const std::string &selected) {
 }
 
 static bool SpeedAESGCMChunk(const EVP_CIPHER *cipher, std::string name,
-                             size_t chunk_byte_len, size_t ad_len) {
+                             size_t chunk_byte_len, size_t ad_len, bool encrypt) {
```

Comment:
any particular reason to measure either encryption or decryption? what's wrong with always measuring both in this function?

Reply:
This was just to arrange the output as grouping the encryption operations for all input sizes together (per key size) then all decryption operations as in the PR description.